ovos_plugin_manager/utils/config.py

```
@@ -144,7 +145,7 @@ def get_plugin_supported_languages(plug_type: PluginTypes) -> dict:
             load_plugin_configs(plug,
                                 PluginConfigTypes(f"{plug_type.value}.config"))
         for lang, config in configs:
```

Ruff

Loop control variable `config` not used within loop body

Rename unused `config` to `_config`

(B007)